test/unit/test_utils.c

```diff
@@ -367,6 +367,73 @@ START_TEST(test_timespec_diff_1ms_overflow)
 }
 END_TEST
 
+
+START_TEST(test_as_key_move_an_empty_rec)
+{
+	as_key* test_key = (as_key *) cf_malloc(sizeof(as_key));
```

Comment:
I see Clayton had defined a `test_malloc` function that is used in other tests. I think we should use that here. I think the memory allocated by these should also be freed in the same tests. We have a common citrusleaf file called alloc.h that defines wrapped versions of these prefixed with `cf_`. I think using `cf_free` would be good. So `test_malloc()` and `cf_free()`.

Reply:
thanks for the info. Could you please check out the order of calling "cf_free" in the latest commit?